src/components/InputItems.js

```diff
@@ -38,24 +38,29 @@ function InputItems(props) {
 
   return (
     <form onSubmit={handleSubmit}>
```

Comment:
- [ ] I kindly suggest that you follow the Zeplin form design by ensuring the following:

- Your `ADD NEW BOOK` title uses the same font style and weight as seen in the Zeplin design given.
- Your inputs have the same style as the Zeplin design given.
- Your ADD BOOK button should be on the same line as the rest of your form, and have the same styles as seen in the Zeplin design.
- Your ADD BOOK button should have no border as seen from the figma design

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="625" alt="image" src="https://user-images.githubusercontent.com/52934128/188274412-f1efe726-5379-4b52-8fd3-7107bf000f9b.png">| <img width="919" alt="image" src="https://user-images.githubusercontent.com/52934128/188274349-264037d7-2c39-49cc-b6fe-7fa9f085d3f1.png"> |